pricejar/base/models.py

```diff
@@ -27,11 +27,12 @@ def __str__(self):
         
 # contact model
 class Contact(models.Model):
-    name = models.CharField(max_length=150)
```

Comment:
I hope you have mademigrations to reflect the chages

Reply:
No I'll quickly do that